src/main/java/com/dani/internapp/models/User.java

```diff
@@ -0,0 +1,43 @@
+package com.dani.internapp.models;
+
+public class User {
+    private String firstname;
+    private String lastname;
+    private String username;
+    private String dob;
+
```

Comment:
- dateOfBirth would sound better than dob. Let your variable name be clear to understand what they stand for.
- dateOfBirth represents a date not a string

Reply:
Thank you, working on it.